.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: "Build and Test"
+
+on: [push, pull_request]
+
+jobs:
+  build_test:
+    strategy:
+      matrix:
+        puzzles_directory: ["./puzzle-1"]
```

Comment:
Does this mean that going forward, once this puzzle is done and dusted, when we have a puzzle-2, we can just rename it to that particular folder so that only that folder is built?

Reply:
Yea that's correct. And you can also have more than one puzzle and just add them to the list.